### PR TITLE
Fix Typescript Lint error basic.

### PR DESCRIPTION
**Pull request by yjaeseok**
I fix typescript lint error.

ISSUE=#467

### CODE REVIEW

**Comment by yjaeseok** [comment]
I request it, but we have more lint error in server folder!

File: tslint.json [hunk 1]
```diff
@@ -1,3 +1,8 @@
 {
-  "extends": "tslint-microsoft-contrib"
+  "extends": "tslint-microsoft-contrib",
+  "rules": {
+	  "no-suspicious-comment": false,
+	  "no-reserved-keywords": false,
+	  "no-relative-imports": false
```
Comment (romandev) [inline tslint.json]: We should fix the issue instead of suppressing this rule. WDYT?

Comment (yjaeseok) [reply tslint.json]: Your means that we fix all lint issue?

Comment (yjaeseok) [reply tslint.json]: @romandev Okay, I remove this ignore rules. :(

**Comment by romandev** [comment]
@yjaeseok, could you update commit description? You seem to remove that ignore cases?
(Patch looks good to me.)

File: tslint.json (hunk 1 above)
Comment (romandev) [reply tslint.json]: I don't understand your previous comments exactly :)
I wanted to fix all lint errors in the future (not in this patch).
If you thought that some rules are unnecessary, we can discuss about it together! (in Slack or here)

Comment (yjaeseok) [reply tslint.json]: Okay, once the ignore rule has been removed,
After that, it would be better to check if there are any unnecessary rules.

**Comment by yjaeseok** [comment]
@romandev Okay, I updated commit description.
Next, we should discuss about ignore rules.

**Pull request merged**